homeassistant/components/tankerkoenig/__init__.py

```diff
@@ -170,14 +170,14 @@ def __init__(
             update_interval=timedelta(minutes=update_interval),
         )
 
-        self._api_key = entry.data[CONF_API_KEY]
-        self._selected_stations = entry.data[CONF_STATIONS]
+        self._api_key: str = entry.data[CONF_API_KEY]
```

Comment:
It's not wrong, but I'm not sure if the type declaration improvements belong in this PR, or a separate PR would be better. In the end 5 months from now, nobody will probably care though 😉

Reply:
🤫

Reply:
tbh, this does not hurt (_no functional changes_), but helps during development, so it should be ok to be in here.

Reply:
Type changes are always beneficial and welcomed.